rasa/shared/core/domain.py

```diff
@@ -832,7 +832,7 @@ def raise_action_not_found_exception(self, action_name_or_text: Text) -> NoRetur
             f"Available actions are: \n{action_names}"
         )
 
-    def random_template_for(self, utter_action: Text) -> Optional[Dict[Text, Any]]:
+    def random_response_for(self, utter_action: Text) -> Optional[Dict[Text, Any]]:
```

Comment:
let's not rename this as it's 1) part of the Rasa API and 2) deprecated and will be removed anyway.

Reply:
alright!